src/main.rs

```diff
@@ -6,11 +6,13 @@ mod rio;
 use crate::rio::application::Application;
 use crate::rio::configuration::{Configuration, ConfigurationError};
 use crate::rio::logging::{Context, FastlyLogger};
+use crate::rio::request_sender::{DirectRequestSender, RequestSender};
 use fastly::{ConfigStore, Error, Request, Response};
 
 #[fastly::main]
 fn main(req: Request) -> Result<Response, Error> {
     let config_store = ConfigStore::open("redirectionio");
+    let req_sender = DirectRequestSender;
```

Comment:
How do you change this sender in your application?

Reply:
we have some something like:
```
pub struct FooAppRequestSender;
impl FastlyRequestManager for FooAppRequestSender {
    fn send(&self, req: Request, backend: String) -> Result<Response, SendError> {
        // custom implementation for the send() method
    }
}
```

and in main.rs:
```
let req_sender = FooAppRequestSender;
```

Reply:
Okay so you edit directly this code 👍🏼